### ISSUE TITLE
issues with color assignment 

### ISSUE THREAD

**Issue by veillette** [turbomaze/JS-Fourier-Image-Analysis#2]
Thanks @turbomaze for your work. 

PhET (https://github.com/phetsims)   has been leveraging some of your code on Fourier Image Analysis to develop an educational open source simulation on wave interference (https://github.com/phetsims/wave-interference).

I'd like to report an issue with the color assignment in main.js

```
 var logOfMaxMag = Math.log(cc*maxMagnitude+1);
    for (var k = 0; k < dims[1]; k++) {
      for (var l = 0; l < dims[0]; l++) {
        var idxInPixels = 4*(dims[0]*k + l);
        currImageData.data[idxInPixels+3] = 255; // full alpha
        var color = Math.log(cc*$h(k, l).magnitude()+1);
        color = Math.round(255*(color/logOfMaxMag));
        // RGB are the same -> gray
        for (var c = 0; c < 3; c++) { // lol c++
          currImageData.data[idxInPixels+c] = color;
        }
      }
```

The line with the color assignment has indices reverted.
If we follow your convention of unpacking the array, then 
the indices for the color assignment should be transposed.
```
  var color = Math.log(cc*$h(l, k).magnitude()+1);
```

**Comment by turbomaze** [comment]
Thank you very much for the fix! I'm glad you're finding this project useful. I've gone ahead and made a commit that makes this fix.

**Comment by waldyrious** [comment]
For future reference, the commit that fixed this issue was ede7772.
